programs/sample_http_link/README.md

```diff
@@ -0,0 +1,49 @@
+#Running Sample HTTP Link
+##Setup Identities
+###Create Identities
+We will create a host and client identity.
+
+For the purposes of this example and for the purpose of brevity we will create the device identities and configure the endpoints all in one command though this can be done separately.
+```
+ziti edge create identity device httpbin.server -a httpbinServerEndpoints -o httpbin.server.jwt
+ziti edge create identity device httpbin.client -a httpbinClientEndpoints -o httpbin.client.jwt
+```
+###Enroll The Identities
+We now have to enroll the identities so the controller knows about them.
+```
+ziti edge enroll httpbin.server.jwt
+ziti edge enroll httpbin.client.jwt
+```
+##Set Up The Service
+### Create A Service Config
+We'll create the service config before the actual service so that we can pass in the config when we create the service. This will save extra steps.
+```
+ziti edge create config httpbin-host.v1 host.v1 '{"protocol":"tcp", "address":"httpbin.org","port":80}'
+```
+###Create A Service
+Create a service to allow access the network traffic.
```

Comment:
creating a service just defines it - really it's the policies that add the 'allow' part... this is just the definition